setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = F403,F821
+select = F,E722
+exclude = __init__.py,sympy/parsing/latex/_antlr/*,sympy/parsing/autolev/_antlr/*,sympy/integrals/rubi/*
```

Comment:
I don't think we should exclude rubi.

Reply:
For some reason I thought there was autogenerated code in there. I'll remove it if not.

Reply:
It also adds around 28,000 lines to the report.

Reply:
It also adds around 13,000 lines to the report.

Reply:
I think parts of it might be but others are not. At any rate, we should aim to have the whole thing pass pyflakes, whether it is autogenerated or not.